syntax = "proto3";

package communicator_objects;

// The initializaiton message - this is typically sent from the Python trainer to the C# environment.
message UnrealRLInitializationInputProto {
    int32 seed = 1;

    // Communication protocol version that the initiating side (typically the Python trainer) is using.
    string communication_version = 2;

    // Package/library version that the initiating side (typically the Python trainer) is using.
    string package_version = 3;

    // The number of training areas to instantiate
    int32 num_areas = 4;
}
